hz: 10             # ループ周期 [Hz]
target_margin: 0.3 # 車両マージン [m]

# --- 膨張モード ---
# mode1: 上下左右に膨張(実行時間: 2.2s)
# mode2: 円形状に膨張  (実行時間: 0.3s)
mode: 2
